components/automate-chef-io/content/docs/configuration.md

```diff
@@ -60,28 +62,29 @@ upgrade_strategy = "at-once"
 
 Then run `chef-automate config patch </path/to/your-file.toml>` to deploy your change.
 
-To upgrade a Chef Automate installation when the `upgrade_strategy` is set to `none`, run:
+To upgrade a Chef Automate installation with the `upgrade_strategy` set to `none`, run:
 
 ```bash
 chef-automate upgrade
 ```
 
-This will upgrade Chef Automate to the latest version from your install channel.
+This command will upgrade Chef Automate to the latest version from your install channel.
 
 #### Deployment Type
 
-Do not change `deployment_type` at this time. Currently, the only supported `deployment_type` is `local`.
+Do not change `deployment_type`. 
+The only supported `deployment_type` is `local`.
 
 #### Settings
 
-Currently, you cannot change the admin username, name, and password set during initial deployment.
+You cannot change the admin username, name, and password set during initial deployment.
 
 To change the admin password after deployment, use the Automate UI.
-Log in as the admin user, navigate to the _User_ page under the **Settings**  tab. Selecting "Local Administrator" opens a form for updating the password. Enter and confirm your new password in the interface, and then select the **Update Password** button to save your changes.
+Log in as the admin user, navigate to the _User_ page under the **Settings** tab.
+Selecting "Local Administrator" opens a form for updating the password.
```

Comment:
this is outdated, after going to the user detail page they now have to go to the reset password tab to reset the password